deployment/setup_20.04.sh

```diff
@@ -848,6 +849,68 @@ function webserver() {
   #TODO(@vn): allow installing nginx only without SSL
 }
 
+
+##############################################################################
+# Report cwctl events to hub
+# Globals:
+#   CHATWOOT_HUB_URL
+# Arguments:
+# event_name: Name of the event to report
+# event_data: Data to report
+# installation_identifier: Installation identifier
+# Outputs:
+#   None
+##############################################################################
+function report_event() {
+  local event_name="$1"
+  local event_data="$2"
+
+  CHATWOOT_HUB_URL="https://hub.2.chatwoot.com/events"
+  
+  # Check if the CHATWOOT_HUB_URL is defined
+  if [ -z "$CHATWOOT_HUB_URL" ]; then
+    echo "CHATWOOT_HUB_URL is not defined. Unable to report event."
+    return 1
+  fi
+
+  # get installation identifier
+  local installation_identifier=$(get_installation_identifier)
+
+  # Prepare the data for the request
+  local data="{\"installation_identifier\":\"$installation_identifier\",\"event_name\":\"$event_name\",\"event_data\":{\"action\":\"$event_data\"}}"
+
+  # Make the curl request to report the event
+  curl -X POST -H "Content-Type: application/json" -d "$data" "$CHATWOOT_HUB_URL" -s -o /dev/null
+}
+
+
+##############################################################################
+# Get installation identifier
+# Globals:
+#   None
+# Arguments:
+#   None
+# Outputs:
+#   installation_identifier
+##############################################################################
+function get_installation_identifier() {
+
+  local installation_identifier
+
+  sudo -i -u chatwoot << "EOF"
+  cd chatwoot
+  installation_identifier=$(bundle exec rake instance_id:get_installation_identifier)
+EOF
+
+  # Check if the installation identifier is defined
+  if [ -z "$installation_identifier" ]; then
+    echo "Installation identifier is not defined. Unable to report event."
```

Comment:
This is a debug log, right? Should we show this to the user?

Reply:
Yes, removed.